.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,84 @@
+name: Build_with_cibuildwheel_and_upload_to_PyPI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - 'MCEq/**'
+      - '!MCEq/data/README.md'
+    tags:        
+      - 'release**' 
+  pull_request:
+    types:
+      - opened
+    branches:    
+      - master
+    paths:
+      - 'MCEq/**'
+      - '!MCEq/data/README.md'
+
+jobs:
+    make_sdist:
+      name: Make SDist
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+
+        - name: Build SDist
+          run: pipx run build --sdist
+
+        - uses: actions/upload-artifact@v3
+          with:
+            path: dist/*.tar.gz
+            
+    build_wheels:
+      name: Wheel on ${{ matrix.os }}
+      runs-on: ${{ matrix.os }}
+      strategy:
+        fail-fast: false
+        matrix:
+          os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+      steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+ 
+      - name: Build wheels
```

Comment:
One more thing here is that we were building the wheels for an entire matrix of python versions from 3.7-3.10. I'm not seeing this happening now. Is this something to worry about?

Reply:
In a new version of the workflow file the matrix of python versions are defined explicitly.